firestore/README.md

```diff
@@ -48,6 +48,7 @@ Run all tests:
 
 Set the `FIRESTORE_PROJECT_ID` environment variable and run the command:
 ```
+   export FIRESTORE_PROJECT_ID=<YOUR-PROJECT-ID>
```

Comment:
This line of code only works for unix operating systems, not for those users on windows which is why the above line leaves it up to the developer to know how to do it for their workstation.